addons/hr_work_entry/models/hr_work_entry.py

```diff
@@ -66,6 +67,52 @@ class HrWorkEntry(models.Model):
     def init(self):
         tools.create_index(self._cr, "hr_work_entry_date_start_date_stop_index", self._table, ["date_start", "date_stop"])
 
+    def _get_actual_durations(self):
+        return sum(work_entry.duration for work_entry in self)
```

Comment:
this helper method is very strange and if I get it correctly, it also does not make sense outside of the context that it is used.
And also the naming is confusing. We should rather remove it

Reply:
Hello @sofiagvaladze 
This helper method is overridden in hr_payroll for part-time calculation